app/views/docs/command-line.phtml

```diff
@@ -138,15 +138,15 @@ brew install --HEAD appwrite</code></pre>
     "collections": [
         {
             "$id": "6213bb2d567d8a206614",
-            "$read": [
-                "team:admin"
-            ],
-            "$write": [
-                "team:admin"
+            [
```

Comment:
Missing the `$permissions` key for the array.

Reply:
is it `$permissions` or `permissions`

Reply:
`$permissions` on response, `permissions` on request params.